zstacklib/zstacklib/utils/pci.py

```
@@ -408,28 +408,68 @@ def parse_resources(device_path):
+            host_domain = source_address.get('domain').replace('0x', '')
+            host_bus = source_address.get('bus').replace('0x', '')
+            host_slot = source_address.get('slot').replace('0x', '')
+            host_function = source_address.get('function').replace('0x', '')
+            host_pci_address = "{}:{}:{}.{}".format(
+                host_domain, host_bus, host_slot, host_function)
+            alias_to_host[alias_name] = host_pci_address
+    
+    # Query actual PCI addresses inside VM
+    try:
+        cmd = "virsh qemu-monitor-command {} --hmp \"info pci\"".format(vm_uuid)
+        r, qemu_output, e = bash_roe(cmd)
```

Ruff (0.14.14)

[warning] Unpacked variable `e` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)